src/plugins/orangehrmBuzzPlugin/Api/Model/BuzzFeedPostModel.php

```diff
@@ -68,7 +68,7 @@
  *     @OA\Property(
  *         property="originalPost",
  *         type="null",
```

Comment:
```suggestion
```
@devishke-orange @ManulMax seems we can remove this right.

Reply:
we can